cita-executor/core/src/cita_executive.rs

```diff
@@ -104,14 +95,19 @@ impl<'a, B: DB + 'static> CitaExecutive<'a, B> {
             return Err(ExecutionError::VM(VMError::NotEnoughBaseGas));
         }
 
-        if t.action == Action::AbiStore && !self.transact_set_abi(&t.data) {
-            return Err(ExecutionError::TransactionMalformed(
-                "Account doesn't exist".to_owned(),
-            ));
+        let mut amend = Amend::new(self.state_provider.clone());
+        if t.action == Action::AbiStore {
```

Comment:
while `AbiStore` is an amend function?

Reply:
It sounds so weird. You are right, separated from amend functionality is better.

Reply:
Please check it again.
`Set abi` is different of `amend abi`

Reply:
Just ensure previous logic, more refactor works later.